src/compat/posix/include/sys/stat.h

```diff
@@ -94,10 +94,12 @@ typedef struct stat {
 //	unsigned  st_mtime;   /* time of last modification */
 //	unsigned  st_ctime;   /* time of last status change */
 
+    struct timespec st_crtim; /* time of create */
```

Comment:
You should use standard fields for standard structures ` struct stat ` contains  
[https://pubs.opengroup.org/onlinepubs/007904875/basedefs/sys/stat.h.html](https://pubs.opengroup.org/onlinepubs/007904875/basedefs/sys/stat.h.html)
The stat structure shall contain at least the following members:
```
time_t st_mtime;   /* time of last modification */
time_t st_ctime;   /* time of last status change */
```